### PR TITLE
Add Cloud Functions implementation for gsuite exporter

### PR DESCRIPTION
**Pull request by Jeanno**
A cloud function implementation of the GSuite log exporter which pull logs from GSuite / Cloud Identity. This implementation uses the pip module `gsuite-exporter`
https://pypi.org/project/gsuite-exporter/

This component includes:
- Deployment-ready Cloud Function code
- Deploy script to deploy Cloud Scheduler, PubSub topic and Cloud Function

### CODE REVIEW

**Comment by Jeanno** [comment]
You mean in the README.md file? Sure.

**Comment by morgante** [comment]
Yeah.

**Comment by Jeanno** [comment]
Tried to fix main.py for the cloud build check but was not successful. The message from cloud build log is not helpful - "Some style in main.py need to be fixed"

File: tools/gsuite-exporter-cloud-function/README.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# README
```
Comment (mik-laj) [inline tools/gsuite-exporter-cloud-function/README.md]: Can you provide descriptive title?

**Comment by jdav-goog** [comment]
Is there any way we can reopen this and I can do any work we need to get it merged in? It would really help us in getting a blog post published. I believe our blocker was "Some style in main.py need to be fixed"

**Comment by jdav-goog** [comment]
Cloud Build is saying "Some files need to be formatted in ./tools/gsuite-exporter-cloud-function - FAIL
./tools/gsuite-exporter-cloud-function/main.py"
Anyone have any insight as to what this means and how we can fix it?

**Comment by jdav-goog** [comment]
Now passing all automated checks. Morgante, is there another change you would still like to see? I believe Jeanno has already made the change to link to the TF module in the README as requested above

**Pull request merged**